docs/1-introduction/1.3.2-vim.md

```diff
@@ -109,6 +109,16 @@ Normal mode.
 
 ?> To learn more about Vim, use the command `vimtutor` in your terminal to run through a tutorial that explores more advanced commands.
 
+Time for a few simple quizzes. Let's start off by making sure you're familiar with the basic mode differences
+<div class="quizdown">
+  <div id="chapter-1/1.3.2/vim-modes-quiz.js" ></div>
+</div>
+
```

Comment:
Feels like these quizzes should be separated as two back to back quizzes looks kind of bad. Maybe we just make one larger quiz. Initially I was thinking to move this one up right under the section on Modes but that would make it too easy I think.

Reply:
I originally wanted the first 4 questions about the modes to take place before the harder questions, but still wanted to randomize the order of the questions. I looked into it, and you can't randomize the question order while still having the same 4 questions take place before the rest. I'll just combine the quizzes and not shuffle the question order

Reply:
Combined 👍